### PR TITLE
Feature newsweek

### PR DESCRIPTION
**Pull request by Ahmed-Benj**
Hello TSE,

In this project, we have cloned the newsweek webpage : 

- Navbar
- Body with following sections : 

1. FEATURED STORIES
2. TOP STORY
3. OPINION
4. THE DEBATE
5. LATEST NEWS
6. IN THE MAGAZINE
7. EDITOR'S PICK
8. SUBSCRIBE
9. NEWSLETTER

- Footer

Could you please review our work on this project ?

Kind regards,
Ahmed

### CODE REVIEW

File: README.md
Comment (teekaytech) [inline README.md]: Please add your live demo link. [this resource](https://raw.githack.com/) might help.

File: assets/css/style.css
Comment (teekaytech) [inline assets/css/style.css]: Since you comment on these blocks of code, it is a good practice to remove them completely.

File: index.html
Comment (teekaytech) [inline index.html]: The `alt` attribute is missing in this `` tag. This is a recurring issue in this file. Kindly add it here and all other places where it might be missing.

File: assets/css/style.css
Comment (teekaytech) [inline assets/css/style.css]: This is a bootstrap project. You should use bootstrap [flex utility classes](https://getbootstrap.com/docs/4.3/utilities/flex/). Kindly fix this and other occurrences of such.

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :construction:

### Hi team :wave: 
So far, this is great work: the progress is promising and I believe that after a couple of changes, the project should be ready. Talking about changes, let us review some of the changes required (see comments below).

**[ REQUIRED CHANGES ]** 

- [x] Although you have a brief description, it does not really summarize what the PR all about. Kindly update your PR description. [This guide](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/) might help in preparing a professional pull request description. 

Source page:


Your clone:



- [x] The date section is missing on your page. Please add it.



- [x] Your footer should contain the Newsweek logo. Kindly add it.

Source page:


Your clone page:


- [x] Since the main point of this project is to use bootstrap for your layout, you are not expected to use `display: flex` or `display: grid` rules in your CSS file. 
- There are bootstrap classes for these rules, please use them. Here is bootstrap references for [flex](https://getbootstrap.com/docs/4.3/utilities/flex/) and [grid](https://getbootstrap.com/docs/4.3/layout/grid/) layouts



- [x] Your footer seems to be missing the pipe symbols `|` separating each item. Please add them all.

Source page:


Your page:



**At 1024px screen size:**

- [x] There seems to be some overlap/overflow in your navbar. Kindly review and fix it.

Source page:


Your page:


- [x] `The Debate` section changes its layout. Please fix it

| Source Page | Your Clone Page |
| ----------- | ----------- |
|  |  |

- [x] Compare your entire page with the source page and take care of the following:
- Font boldness, paddings, and margins
- Image responsiveness
- White spaces between each section (e.g. space between `In the magazine` and `Editors' Picks` section.

- [x] Please check the comments I made in the code section below and do the needful.

**[ OPTIONAL CHANGES ]**

- [x] You are advised to use semantic HTML always. Semantic HTML5 tags like `<header>`, <main>`, `<article>`, <section>` etc should be used as appropriate in your project.

- [x] Linter files are present in your project, but they are not executed by GitHub Actions. Since you already set them up, you should find a way to make the checks run on your PR.

**[ COMMENTS ]**

- [x] I observed that in a couple of places in your style.css file, you used the `!important` rule often. Although this might get the work done, you should be careful not to overuse the rule as it might mess things up for you in the future.

- [ ] Also, I observed that in your index.html file, you used a lot of SVG images directly. I'm not against using SVG images, but personally, I think it's better to have them stored in your assets folder, then you reference them in your HTML file just like images. This makes your code cleaner and readable.

* **When you have made the specified changes, please submit another code review. Thank you!**
* **If you feel like you need help or you have any question(s), I'm @taofeekolalere on slack**

Regards,
[Taofeek Olalere](https://github.com/teekaytech) (@taofeekolalere on slack)
Happy Coding :technologist:

File: README.md
Comment (Ahmed-Benj) [reply README.md]: Done with netlify

File: assets/css/style.css
Comment (Ahmed-Benj) [reply assets/css/style.css]: Done

File: assets/css/style.css
Comment (Ahmed-Benj) [reply assets/css/style.css]: removed all unnecessary occurences of flex in css and replaced them by bootstrap classes d-flex

**Comment by Ahmed-Benj** [comment]
Hello Sir,

I have fixed all requested changes, could you pleease approve my project ?

Kind regards,
Ahmed

**Comment by Ahmed-Benj** [comment]
Fixed all of them

File: index.html
Comment (Ahmed-Benj) [reply index.html]: Fixed all the items

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello team :wave:

__I must say, you've done an amazing fixing those required changes in the last review. However, this project is still not complete, therefore I cannot approve it. Let's quickly fix the changes identified and re-submit another review request.__

**[ REQUIRED CHANGES ]**

_Do not be discouraged. This is a big project and most times, some changes may lead to another. Also, we need to ensure that nothing is messed up with the page layout across different breakpoints._

- [x] Since the main point of this project is to use Bootstrap, kindly use more bootstrap classes for features like padding, margin, width, color, etc. You're using more CSS rules than bootstrap classes. Try to study bootstrap more so that you can fully utilize its classes.

- [x] Your live demo link still presents the old version of the project. Kindly ensure that your changes are being tracked by Netlify so that it renders your updated project. As an option, you can use [githack](http://rawcdn.githack.com/).

- [ ] Your linter checks are not running. You should ensure that GitHub Actions checks your linters and no error is returned.

- [x] Your navbar items are not well-spaced. Kindly fix it.

Original page:


Your page:


- [x] You have some missing components/sections and layout issues.
- Your content is too condensed without space. Please check the image below for a better understanding
- `The Debate` section is missing on your page. Kindly re-add it.
- Ensure that each article's title is well distinguished from the summary. Use a bigger font size and bold to make it similar to the original page.

Original Page:


Your Page:


- [x] Add a little space between the `In the Magazine` and `Editor's Pick` section.

- [ ] The `My Turn` and `Culture & Travel` section is missing. Please add those sections.

- [x] The red Newsweek logo is missing in your footer. Kindly add it.

**At medium screen sizes**

- [x] The date on the top navbar should be hidden.

**At small screen sizes** <768px

- [x] Try to check the original page and make sure that the contents of your page look similar to what we have there. See the images below for some instances.

| Original Page | Your clone page |
| --------------- | ------------------- |
|  |  |
|  |  |


**[ COMMENTS ]**
- Please take some time to carefully compare your page with the original page. Then, make sure that your page looks similar to the source page.
- In this project, you have to be more precise. Apply all texts and images formatting as seen on the original page. 


**Please make the required changes and submit another review request when you are done.**


* * *

**_You can reach out to me on twitter (@ola_lere), Github(@teekaytech), or slack (@taofeekolalere )_**
__Happy Coding :smiley:__

File: index.html
Comment (descholar-ceo) [inline index.html]: Please remove this commented codes

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Change Required :vertical_traffic_light: :warning: 
## Hello @Ahmed-Benj and @Ramon-Carrillo  :handshake: :handshake: :handshake: 
:+1: This is good to make it this far, your clone looks like the original page and it is really nice, congratulations on the good work you have done so far.
:+1: I liked that you have tried to implemented the requested changes in the previous reviews, keep it up, and your project really has a great progress.
## :warning: However we need to make few changes to mark this project approved, don't feel discouraged because of being asked some changes more and more, that's beacuse this is a learning process, and we are here to help you to become an out-standing developer. See the requested change below :point_down: :point_down: :point_down: 
### Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# Required Changes
- [x] Sorry for that it was not mentioned earlier baut your clone has some extra-space on each side, which is different than the original page
Your clone:
Screenshot from 2020-10-24 01-57-23

Original page: 
Screenshot from 2020-10-24 01-57-56
- [x] In the navbar, there should be some dummy weather data above of datetime section
Your clone : 
Screenshot from 2020-10-24 01-59-35
Original page: 
Screenshot from 2020-10-24 02-00-09
- [x] In options bar, the search input should be input type text, it's not a link
- [x] Instead of using US as a title use US, business, world, culture, tech & science, sports
- [x] Make sure that your linters are run on github actions
- [x] Fix all the comments I added directly in your codes

# Optional changes
- [x] Your commit messages should be in imperative mode, something like modify to solve linters reviews in css code instead of modified to solve linters reviews in css code, refer [to this article for mor details](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

If anything seems to be unclear to you, don't hesitate to contact me for more clarification slack: `@Mugirase Emmanuel` | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)

File: index.html
Comment (Ahmed-Benj) [reply index.html]: Done

**Review by jubaan — APPROVED** [review]
# :tada: :tada: :tada: :rotating_light: Status: Approved :rotating_light:  :tada: :tada: :tada:

#### :clap: :clap: Great job :hammer_and_pick:!! There are no more issues left to be addressed. All changes required by previous TSE are done.

## See you at the top :rocket:!

# :tada: :tada: :tada: :rotating_light: Status: Approved :rotating_light:  :tada: :tada: :tada:

**Pull request merged**
